apps/vr-tests/src/stories/AccordionConverged.stories.tsx

```diff
@@ -7,7 +7,7 @@ import {
   AccordionHeader,
   AccordionPanel,
 } from '@fluentui/react-accordion';
-import { CircleRingIcon } from '@fluentui/react-icons-mdl2';
+import { Circle20Regular } from '@fluentui/react-icons';
```

Comment:
might be better like this since, we'll have to move to unsize icons at some point.

```suggestion
import { Circle20Regular as CircleRingIcon } from '@fluentui/react-icons';
```